react-native-google-cast.podspec

```diff
@@ -25,6 +25,11 @@ Pod::Spec.new do |s|
     ss.dependency 'google-cast-sdk'
   end
 
+  s.subspec 'GuestModeArm' do |ss|
+    ss.dependency "#{package['name']}/RNGoogleCast"
+    ss.dependency 'google-cast-sdk-dynamic-xcframework'
```

Comment:
Did you make the pod? It needs to be built and published on cocoapods.

Reply:
oh weird the CocoaPods website didn't find it 🤷‍♂️

Reply:
Yeah, I know. It seems like the indexing is broken for a while.